protected/humhub/modules/admin/resources/js/humhub.admin.PendingRegistrations.js

```diff
@@ -0,0 +1,65 @@
+humhub.module('admin.PendingRegistrations', function(module, require, $) {
+    var object = require('util').object;
+    var Widget = require('ui.widget').Widget;
+    var client = require('client');
+    var urlDeleteSelected;
+
+    var PendingRegistrations = function(node, options) {
```

Comment:
Just use:

```
var PendingRegistrations = Widget.extend();
```

and remove the following line:

```
object.inherits(PendingRegistrations, Widget);
```

Reply:
done!